services/notifier.js

```diff
@@ -17,5 +18,9 @@ module.exports = {
         for (platform in rooms) {
             notifiers[platform].sendMessage(rooms[platform], from, msg, color);
         }
+    },
+
+    sendSlackPersonalMessage: function(userEmail, from, msg, color) {
```

Comment:
Why not use `sendMessage()`?
See: https://github.com/fiverr/devflow/blob/35d7fb532e5a4d70b15009037322661225246be4/jobs/timeNearlyUpReminder.js#L21